### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by mzegla**
### 🧪 Checklist

- [ ] Unit tests added.
- [ ] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

**Review by atobiszei — APPROVED** [review]

File: docs/llm/quickstart.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ node: {
     }
   }
 }
-' > TinyLlama-1.1B-Chat-v1.0/graph.pbtxt
+' >> TinyLlama-1.1B-Chat-v1.0/graph.pbtxt
```
Comment (dtrawins) [inline docs/llm/quickstart.md:55]: why appending?

Comment (mzegla) [reply docs/llm/quickstart.md:55]: still working on going through the CI
manually it worked before and works after that change, so doesn't make a difference

**Review by atobiszei — APPROVED** [review]

File: docs/llm/quickstart.md (hunk 1 above)
Comment (dtrawins) [reply docs/llm/quickstart.md:55]: with appending we can not run this command twice

File: docs/llm/quickstart.md [hunk 2]
```diff
@@ -113,28 +113,28 @@ curl -s http://localhost:8000/v3/chat/completions \
...
 **Note:** If you want to get the response chunks streamed back as they are generated change `stream` parameter in the request to `true`.
 
 
-## References:
-- [Efficient LLM Serving - reference](./reference.md)
-- [Chat Completions API](./model_server_rest_api_chat.md)
-- [Completions API](./model_server_rest_api_completions.md)
-- [Demo with Llama3 serving](./../demos/continuous_batching/)
+## References
+- [Efficient LLM Serving - reference](../reference.md)
```
Comment (dtrawins) [inline docs/llm/quickstart.md:137]: it looks incorrect. reference.md is in the same folder

**Review by dtrawins — APPROVED** [review]

**Pull request merged**